src/Controller/MessagePrivateController.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace Drupal\message_private\Controller;
+
+use Drupal\Core\Url;
+use Drupal\user\entity\User;
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\message\Entity\Message;
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+use Drupal\message\MessageTemplateInterface;
+
+/**
+ * Controller for viewing private messages.
+ */
+class MessagePrivateController extends ControllerBase implements ContainerInjectionInterface {
+
+  /**
+   * The access handler object.
+   *
+   * @var \Drupal\Core\Entity\EntityAccessControlHandlerInterface
+   */
+  private $accessHandler;
+
+  /**
+   * The entity manager service.
+   *
+   * @var \Drupal\Core\Entity\EntityManagerInterface
+   */
+  protected $entityManager;
+
+  /**
+   * Constructs a MessageUiController object.
+   */
+  public function __construct() {
+    $this->entityManager = \Drupal::entityManager();
+    $this->accessHandler = \Drupal::entityManager()->getAccessControlHandler('message');
+  }
+
+  /**
+   * Generates output of all messages sent to the current user
+   *
+   *
+   * @return
+   *   A render array for a list of the messages;
+   */
+  public function inBox($user) {
+    $messages = array();
+
+    // Get messages sent to the current user
```

Comment:
This seems like a stray comment, is it about the whole method, or something else. Either above the code in question, or in the block comment about the method.